### PR TITLE
Updates to Zipkin v2

### PR DESCRIPTION
**Pull request by codefromthecrypt**
tested against https://github.com/spring-cloud/spring-cloud-sleuth/pull/720

### CODE REVIEW

**Comment by adriancole** [comment]
screen shot 2018-04-27 at 12 43 04 pm

File: pom.xml [hunk 1]
```diff
@@ -13,45 +13,41 @@
   <description>Example using Spring Cloud Sleuth to trace RPCs from Spring Web MVC</description>
 
   <properties>
-    <sleuth.version>1.2.4.RELEASE</sleuth.version>
+    <maven.compiler.target>1.7</maven.compiler.target>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <sleuth.version>1.3.0.BUILD-SNAPSHOT</sleuth.version>
```
Comment (marcingrzejszczak) [inline pom.xml:18]: https://github.com/spring-cloud/spring-cloud-sleuth/releases

`1.3.4.BUILD-SNAPSHOT` I guess?

File: pom.xml [hunk 2]
```diff
@@ -13,45 +13,41 @@
...
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-web</artifactId>
-      <version>1.5.4.RELEASE</version>
-    </dependency>
-    <!-- Micrometer automatically adds request metrics when in the classpath -->
-    <dependency>
-      <groupId>io.micrometer</groupId>
-      <artifactId>micrometer-spring-legacy</artifactId>
-      <version>0.10.0.RELEASE</version>
+      <version>1.5.7.RELEASE</version>
```
Comment (marcingrzejszczak) [inline pom.xml:37]: `1.5.12.RELEASE` ?

**Comment by codefromthecrypt** [comment]
very old pr ;) closed it

**Pull request closed without merging**